localstack/services/kinesis/kinesis_listener.py

```diff
@@ -85,7 +85,9 @@ def return_response(self, method, path, data, headers, response):
         elif action == ACTION_PUT_RECORD:
             response_body = json.loads(to_str(response.content))
             event_record = {
+                'approximateArrivalTimestamp': datetime.now(),
```

Comment:
I think we can use `timestamp_millis()` from `utils/common.py` here and change this to:
```
...
'approximateArrivalTimestamp': timestamp_millis(),
```
Same in line 106 below.

At the top of the file we can add the import:
```
from localstack.utils.common import ..., timestamp_millis
```

Reply:
Towards the top of the file, there is another usage for datetime.now(). Should I replace that usage as well?

Reply:
It seems I'm unable to link to lines on mobile, but I'm referring to the forward_request() function where we also use datetime.now().

Reply:
Good catch, would be good to unify the way we're handling timestamps. Note that we're using `json_safe(..)` in `forward_request(..)` above, which transforms the object to be safe for JSON dumping.

I think we can either:
* Standardize on using `timestamp_millis()` instead of `datetime.now()` in the entire file
or
* Use `json_safe(..)` for safe JSON dumping after the code changes you made in the PR so far.

Reply:
Perfect. I standardized usage of `timestamp_millis()`. Let me know if you need anything else!